index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Project</title>
+    <link rel="stylesheet" href="style.css">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600&family=Roboto:ital,wght@0,100;0,400;0,500;0,700;0,900;1,100;1,400;1,500&display=swap');
+    </style>
+  </head>
+  <body>
+
+
+
+<section class="full-page">
+      
+<!------- Navbar & background video --------->
+  
+  <header>
+        <video class="back-video" muted loop autoplay controls playsinline>
+            <source src="videos/Video3.mp4" type="video/mp4">
+        </video>
+
+          <nav>
+              <img src="Images/Earth.png" class="logo">
+              
+              <ul class="nav-links">
+                <a href="#accordion" class="button-faq">FAQs</a>
+              </ul>
+          </nav>
+      </header>
+
+
+<!------- Content over video --------->
+  <div class="content">
+    <h1>Alternative Sports</h1>
+    <p>Want to register a new sport? <br> Create an account and start the journey.</p>
+    <a href="#" class="button" id="button">Login</a>
+    
+
+  </div>
+
+
+
+<!------- Popup window for Login and register forms --------->
+
+  <div class="popup">
+    <div class="popup-content">
+      <img src="Images/close.png" alt="Close" class="close">
+
+<!------- Toggle button - Login/Register --------->
+      
+      <div class="button-box">
+        <div id="btn"></div>
+        <button type="button" class="toggle-btn" onclick="login()">Login</button>
+        <button type="button" class="toggle-btn" onclick="register()">Register</button>
+        </div>
+
+<!------- Login form --------->
+
+        <form id="login" class="input-group" form action="https://httpbin.org/anything" method="post">
+          <input type="text" name="message" class="input-field" placeholder="Username" required>
+          <input type="password" name="message" class="input-field" placeholder="Enter password" required>
+          <input type="checkbox" name="checkbox" class="check-box" required>
+          <label for="checkbox">Remember this device</label>
```

Comment:
I would give different "names" for your input fields. I think it's easier to read the input data (in https-bin) if you have different "headings" for the different inputs. So you easy can connect the data to the input field basically :)